Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -1005,6 +1005,33 @@ final class CodeActionTests: XCTestCase {
       []
     }
   }
+  
+  func testConvertStringConcatenationToStringInterpolation() async throws {
+    try await assertCodeActions(
+      """
+      1️⃣#"["# + 2️⃣key + ": \\(3️⃣d) " + 4️⃣value + ##"]"##5️⃣
+      """,
+      ranges: [("1️⃣", "2️⃣"), ("3️⃣", "4️⃣"), ("1️⃣", "5️⃣")],
+      exhaustive: false
+    ) { uri, positions in
+      [
+        CodeAction(
+          title: "Convert String Concatenation to String Interpolation",
+          kind: .refactorInline,
+          edit: WorkspaceEdit(
+            changes: [
+              uri: [
+                TextEdit(
+                  range: positions["1️⃣"]..<positions["5️⃣"],
+                  newText: "##\"[\\##(key): \\##(d) \\##(value)]\"##"
+                )
+              ]
+            ]
+          )
+        )
+      ]
+    }
```

Comment:
Could you also add a test case to check that we don’t show this refactoring when there’s a standalone string literal.